Firmware/mmu2.cpp

```diff
@@ -240,6 +240,7 @@ bool MMU2::VerifyFilamentEnteredPTFE() {
     uint8_t fsensorState = 0;
     uint8_t fsensorStateLCD = 0;
     uint8_t lcd_cursor_col = 0;
+    fsensor.clearEvent(Filament_sensor::Events::jam);
```

Comment:
Does it make sense to guard this with `#if defined(FILAMENT_SENSOR) && (FILAMENT_SENSOR_TYPE == FSENSOR_PAT9125)`? Seems to save memory on MK3S+